src/atomic-components/Star/Star.jsx

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
```

Comment:
please do not ignore eslint warnings (read more about accessibility) and use semantically correct HTML elements